.github/workflows/release.yml

```diff
@@ -14,17 +14,31 @@ jobs:
         - operating-system: ubuntu-18.04
         - operating-system: windows-2019
           arch: -386
+          ext: ".exe"
         - operating-system: windows-2019
           arch: -amd64
+          ext: ".exe"
         - operating-system: macos-10.15
 
     runs-on: ${{ matrix.operating-system }}
+    env:
+      TARGET: "/CreateBridge/"
+      AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+      AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
 
     steps:
       - name: Set version
         run: echo "TAG_VERSION=${GITHUB_REF##*/}" >> $GITHUB_ENV
         shell: bash
 
+    - name: Identify Prerelease
+        # This is a workaround while waiting for create-release action to implement auto pre-release based on tag
+        id: prerelease
+        run: |
+          wget -q -P /tmp https://github.com/fsaintjacques/semver-tool/archive/3.1.0.zip
+          unzip -p /tmp/3.1.0.zip semver-tool-3.1.0/src/semver >/tmp/semver && chmod +x /tmp/semver
+          if [[ $(/tmp/semver get prerel ${GITHUB_REF/refs\/tags\//}) ]]; then echo "::set-output name=IS_PRE::true"; fi
+
```

Comment:
Step required because we do not want to push auto-update files in case of a pre-release